packages/teams-js/src/private/files.ts

```diff
@@ -431,7 +431,7 @@ export namespace files {
     /**
      * The file path to uniquely identify it within the file hierarchy
      */
-    filePath: string;
+    webkitRelativePath?: string;
```

Comment:
Given that this is renaming a field of an exported type that isn't marked with beta, how are you planning on making this not break existing users of this interface?

Reply:
The feature is in development phase and have not been released for anyone, so no one is using this **File** interface and setting **filePath** field right now. In the process of consuming these changes we realised that **filePath** field will be problematic and hence came back to address it here in SDK.

Reply:
I think that given the offline conversation about the usage of this interface and the recency and the fact that it's private, this change should be okay. However, we should add @beta tag to this interface and any other interfaces or APIs that are in active development to ensure you can continue to adjust the API until it's customer-ready.

Reply:
@TrevorJoelHarris please correct me if you disagree.

Reply:
Applied @beta tag to the upload API. From now onwards we will definitely follow the practice of marking the API / interfaces beta until they become fully customer ready.